### ISSUE TITLE
Can parse workflow object to XML ?

### ISSUE THREAD

**Issue by denygame** [phpmentors-jp/workflower#60]
Hi all, i'm render 1 workflow object, how to be parse it to XML ?

**Comment by pakapakakkkk** [comment]
Yup, i also will know how to parse Workflow Object To File Bpmn 2.0

**Comment by iteman** [comment]
It's possible to transform the Workflow object back to a BPMN file, but for the purpose of displaying and editing BPMN graphs in a Web application, it is easy to draw the original BPMN file using a tool like bpmn-js.

**Comment by vesper8** [comment]
@iteman 

Sorry to bump up an old thread. You say it's possible to transform the Workflow object back to a BPMN file. But I can't find any exporting mechanism in the library.. could you give me a hint as to how I would go about exporting to a .bpmn file that I could then load up using a tool like bpmn-js ?

Is this part of the library and I just haven't found it or .. ? If you have any code you could share that would be so appreciated. Many thanks!
